docs/content/staking/index.mdx

```diff
@@ -6,15 +6,16 @@ description: Introduction to how staking works on Flow
 
 This documents provides an introduction to staking, an important action that
 users of a proof of stake (PoS) blockchain need to be aware of to ensure the security
-of the network and receive token rewards.
+of the network and to receive token rewards.
 
 # What is Staking?
 
 A blockchain is a global network of thousands of computers who work together
 to maintain the security and integrity of a shared state that millions of applications and people around the world use. 
 Every node operator has a small part of the responsibility to keep the network running smoothly 
 and to ensure that other node operators are doing the same. 
-This large distribution of resources makes it so that control never gets centralized in the control of one person or organization.
+This large distribution of resources makes it so that control never gets centralized 
+in the control of one person or organization.
```

Comment:
```suggestion
This large distribution of resources makes it so that control is never centralized 
under one person or organization.
```

"Control" was mentioned twice, lmk if this wording is OK